pyproject.toml

```diff
@@ -12,11 +12,12 @@ authors = [
 description = "Extend NeXus for materials science experiment and serve as a NOMAD parser implementation for NeXus."
```

Comment:
Extend NeXus for experiments and characterization in Materials Science and Materials Engineering ...

Reply:
```suggestion
description = "Extend NeXus for experiments and characterization in Materials Science and Materials Engineering and serve as a NOMAD parser implementation for NeXus."
```